ci/axis/base-runtime.yaml

```diff
@@ -11,9 +11,7 @@ IMAGE_TYPE:
 
 RAPIDS_VER:
   - 0.16
-
-RAPIDS_CHANNEL:
```

Comment:
We use `RAPIDS_CHANNEL` when we do a release, as we need to switch to `rapidsai` from `rapidsai-nightly`